src/components/overlays/chat/ChatWindow.vue

```diff
@@ -0,0 +1,132 @@
+<!--
+//  ChatWindow.vue
+//
+//  Created by Kalila L. on May 13th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+
+<style lang="scss" scoped>
+    .q-field {
+        background-color: rgba(0, 0, 0, 0.4);
+    }
+</style>
+
+<template>
+    <OverlayShell
+        icon="chat"
+        title="Chat"
+        :defaultHeight="300"
+        :defaultWidth="600"
+        :defaultLeft="0"
+        :hoverShowBar="true"
+        :style="{ 'background': 'transparent', 'box-shadow': 'none', border: 'none' }"
+    >
+        <q-card
+            class="column no-wrap items-stretch full-height"
+            style="background: transparent; box-shadow: none;"
+        >
+            <q-scroll-area
+                class="col"
+                style="height: 100%"
+            >
+                <q-card-section class="q-pt-none">
+                    <div v-for="message in worldChatHistory" :key="message.timestamp">
+                        <q-chat-message
+                            :text="[ message.message ]"
+                            :sent="message.self"
+                            :name="message.displayName"
+                            :stamp="message.timestamp"
+                            :avatar="getProflePicture(message.username)"
+                            text-color="white"
+                            bg-color="primary"
+                        />
+                    </div>
+                </q-card-section>
+                <!-- <q-card-actions align="right">
+                    <q-btn v-close-popup flat color="primary" label="Reserve" />
+                    <q-btn v-close-popup flat color="primary" round icon="event" />
+                </q-card-actions> -->
```

Comment:
Delete, or add a comment saying why temporarily commented out.

Reply:
I like to keep stuff around that I don't often know how to do so I can quickly use it if I ever have the need. But in this case that's quite simple so I'll get red of it.